search/find-place/src/test/java/com/esri/samples/find_place/FindPlaceTest.java

```diff
@@ -0,0 +1,206 @@
+package com.esri.samples.find_place;
+
+import com.esri.arcgisruntime.geometry.Point;
+import com.esri.arcgisruntime.mapping.view.Callout;
+import com.esri.arcgisruntime.mapping.view.Graphic;
+import com.esri.arcgisruntime.mapping.view.GraphicsOverlay;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import javafx.geometry.Point2D;
+import javafx.geometry.Pos;
+import javafx.geometry.VerticalDirection;
+import javafx.scene.Node;
+import javafx.scene.control.Button;
+import javafx.scene.control.ComboBox;
+import javafx.scene.control.Label;
+import javafx.scene.input.MouseButton;
+import javafx.scene.layout.Pane;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.testfx.api.FxRobot;
+import org.testfx.api.FxToolkit;
+import org.testfx.framework.junit.ApplicationTest;
+import org.testfx.robot.Motion;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+public class FindPlaceTest extends FxRobot {
+
+  public static final int SLEEP_500MS = 500;
+  public static final int SLEEP_1000MS = 1000;
+  public static final int SLEEP_1500MS = 1500;
+
+  MapView mapView;
+  GraphicsOverlay graphicsOverlay;
+  ComboBox<String> placeBox;
+  ComboBox<String> locationBox;
+  Node locationBoxArrowRegion;
+  Node placeBoxArrowRegion;
+  Button searchButton;
+
+  @Before
+  public void setup() throws Exception {
+    // launch the sample
+    ApplicationTest.launch(FindPlaceSample.class);
+
+    // get a handle on the MapView and GraphicsOverlay
+    Node mapViewNode = lookup("#mapView").query();
+    if (mapViewNode instanceof MapView) {
+      mapView = (MapView) mapViewNode;
+      graphicsOverlay = mapView.getGraphicsOverlays().get(0);
+
+    } else {
+      fail("MapView Node could not be found");
+    }
+
+    // get a handle on the ComboBoxes
+    placeBox = lookup("#placeBox").queryComboBox();
+    locationBox = lookup("#locationBox").queryComboBox();
+
+    // get a handle on the ComboBoxes' arrow buttons
+    for (Node child : locationBox.getChildrenUnmodifiable()) {
+      if (child.getStyleClass().contains("arrow-button")) {
+        locationBoxArrowRegion = ((Pane) child).getChildren().get(0);
```

Comment:
Since there's two ComboBoxes, I needed to find a way to get a handle on each 'arrow-button' and know which is which. Since I think if I simply do `lookup(".arrow-button").queryAll()` I'll get two nodes and not know which ComboBox they belong to? Might be a more elegant way to do this though